docker/docker-compose-dev.yml

```diff
@@ -55,15 +55,17 @@ services:
     networks:
       - backend
 
-  redisearch:
+  q_yaar_redisearch:
     image: redislabs/redisearch:latest
     ports:
-      - "6379:6379"
+      - "6380:6380"
```

Comment:
Using 6380 to not interfere with 6379 (default port) running for my 1Bridge codebase.